imap_processing/tests/idex/test_idex_l1a.py

```diff
@@ -292,27 +292,35 @@ def test_decode_sub_frame_psel_3():
     assert ints == [1, 2, 4, 1, 5]
 
 
-def test_cdf_creation_catlst(decom_test_data_catlst: xr.Dataset):
-    """Verify that a sample of the data can be written to a cdf without errors.
+def test_catlst_dataset(decom_test_data_catlst: xr.Dataset):
+    """Verify that the dataset contains what we expect and can be written to a cdf.
 
     Parameters
     ----------
     decom_test_data_catlst : xarray.Dataset
         The dataset to test with
     """
+    assert "shcoarse" in decom_test_data_catlst
+    assert "shfine" in decom_test_data_catlst
```

Comment:
Can you also assert on an expected data value? The new function you split out isn't tested individually from what I can see.